test/generateMongoidTypes.test.ts

```diff
@@ -0,0 +1,41 @@
+import { exec } from 'child_process';
+import fs from 'fs';
+import { mongoidPrimitiveTypes } from '../src/model/dataTypes/primitiveDataTypes';
+import parser from '../src/parser';
+
+describe('generateMongoidTypes', () => {
+
+  const modelDir = './generated/mongoid';
+
+  // remove models after all tests have been run
+  afterAll(() => {
+    exec(`rm -rf ${modelDir}`);
+  });
+
+  function generateMongoids(): Promise<{ code: number, error: any }> {
+    return new Promise(resolve => {
```

Comment:
I don't think we need to use "exec" here- can't we just call the function normally?

Reply:
it's a script so I need to execute it the way package.json executes it.